src/plugin.json

```diff
@@ -17,7 +17,7 @@
       {"name": "GitHub", "url": "https://github.com/fw-dev/piechart-panel"}
     ],
     "screenshots": [],
-    "version": "%VERSION%",
+    "version": "1.0.2",
```

Comment:
%VERSION% caused the Grafana plugins repository CI to fail, because it's trying to match the version number from this file in our repository

Reply:
Perhaps also %TODAY%, but I'm not sure cause it didn't get that far. I think this could also be fixed by committing the dist folder, which also has this file with the version number and date being added. I've read that it will prioritise that file when a dist folder is included, otherwise it will look at this one in the src folder.